src/test/app/common/calculateInterest.ts

```diff
@@ -4,7 +4,7 @@ import * as moment from 'moment'
 import { calculateInterest } from 'app/common/calculateInterest'
 import { Interest, InterestType } from 'claim/form/models/interest'
 
-describe('calculateInterest', () => {
+xdescribe('calculateInterest', () => {
```

Comment:
imo better to skip the hook with `--no-verify` if you expect tests to fail than skip them in a PR to prevent it actually being merged and also to prevent people asking every time did you mean to do this